index.html

```diff
@@ -94,30 +91,14 @@ <h2><dfn>UnidirectionalStreamsTransport</dfn> Mixin</h2>
     <pre class="idl">
 interface mixin UnidirectionalStreamsTransport {
   Promise&lt;SendStream&gt; createSendStream (optional SendStreamParameters parameters);
-  attribute EventHandler    onreceivestream;
+  ReadableStream            receiveStreams ();
```

Comment:
It could, but that would preclude us from ever being able to add a parameters to it.  I'm not sure what parameters we would add, but we might come up with something.  Do you think an attribute would be much easier to use?

Reply:
Not sure I understand this.  Is the idea that `receiveStreams().getReader.read()` returns the most recently created `receiveStream`s?  Wouldn't an event be more efficient than having to poll?

Reply:
Yes, and I changed the steps based on Steve's comments. Take a look at the latest.